src/main/java/com/firebolt/jdbc/connection/settings/FireboltSessionProperty.java

```diff
@@ -67,6 +67,8 @@ public enum FireboltSessionProperty {
 	USER_CLIENTS("user_clients", null, String.class, "user clients", FireboltProperties::getUserClients),
 	ACCESS_TOKEN("access_token", null, String.class, "access token", p -> "***"),
 	ENVIRONMENT("environment", "app", String.class, "Firebolt environment", FireboltProperties::getEnvironment, "env"),
+	VALIDATE_ON_SYSTEM_ENGINE("validate_on_system_engine", false, Boolean.class,
+			"Whether to validate the connection on the system engine or not. Not validated by default", FireboltProperties::isValidateOnSystemEngine),
```

Comment:
I'd reword this to avoid confusion, otherwise it sounds like we're not doing validation at all. `"Whether to validate the connection on the system engine or not. By default validates on an engine currently connected. "`  or something similar.